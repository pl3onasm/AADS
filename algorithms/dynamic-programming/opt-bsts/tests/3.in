.1 .02 .3 .01 .02 .4 .05 0.03 .01 .03