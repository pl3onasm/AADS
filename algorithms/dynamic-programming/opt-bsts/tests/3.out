OBST cost: 1.82

OBST structure:

--| K0
---| K1
-| K2
---| K3
--| K4
K5
--| K6
-| K7
---| K8
--| K9
